Library/Homebrew/livecheck/strategy/extract_plist.rb

```diff
@@ -102,9 +102,16 @@ def self.find_versions(cask:, url: nil, regex: nil, **_unused, &block)
 
           unversioned_cask_checker = if url.present? && url != cask.url.to_s
             # Create a copy of the `cask` that uses the `livecheck` block URL
-            cask_copy = Cask::CaskLoader.load(cask.sourcefile_path)
-            cask_copy.allow_reassignment = true
-            cask_copy.url { url }
```

Comment:
Can you not just do `cask_copy.url url`?

Reply:
I tried that first.

```
Error: google-earth-pro: wrong number of arguments (given 1, expected 0)
/opt/homebrew/Library/Homebrew/cask/cask.rb:121:in `block (2 levels) in <class:Cask>'
/opt/homebrew/Library/Homebrew/livecheck/strategy/extract_plist.rb:107:in `find_versions'
```

Reply:
I think that's because we define the DSL methods without forwarding arguments here:

https://github.com/Homebrew/brew/blob/3b26dd859d31886c71839bf87dca5ee9bb450efb/Library/Homebrew/cask/cask.rb#L121